pdns/dnsdist.cc

```diff
@@ -894,8 +895,18 @@ shared_ptr<DownstreamState> chashed(const NumberedServerVector& servers, const D
   unsigned int min = std::numeric_limits<unsigned int>::max();
   shared_ptr<DownstreamState> ret = nullptr, first = nullptr;
 
+  double targetLoad = std::numeric_limits<double>::max();
+  if (g_consistentHashBalancingFactor != 0) {
```

Comment:
Maybe verify it is `> 0`, or validate `setConsistentHashingBalancingFactor()` input parameter ?